### DISCUSSION TITLE
Modeliser des processus avec Mercator (BPM) ?

### DISCUSSION THREAD

**Discussion by lynch4fr** [dbarzin/mercator#970] [Ideas]
Bonjour,
Apres avoir fait un inventaire de plusieurs Taches, Opérations et Activités. On serait desormais tenté de vouloir modeliser les processus metiers associés (BPM)   :sunglasses:  Est ce que ça serait une trajectoire de l'outil Mercator ?
_C'est à dire produire des diagrammes BPM (sous la forme de l'explorateur actuel) avec des "evenements, branchement, connecteurs, groupements, etapes..."_ 
C'est juste un idée, merci.
A+

**Comment by dbarzin** [comment]
Merci pour votre suggestion et votre enthousiasme pour Mercator ! 😊

Vous avez touché un point très intéressant : l'évolution vers la modélisation des processus métiers (BPM) est effectivement une trajectoire majeure envisagée pour Mercator. Nous prévoyons de développer un outil dédié au dessin d'architectures et de diagrammes BPM.

Ce développement est une priorité et nous espérons pouvoir proposer cette fonctionnalité dès l'année prochaine. Nous vous tiendrons informé des avancées !

**Comment by lynch4fr** [comment]
Ça fait plaisir d'avoir un logiciel comme Mercator qui évolue et toujours aussi proche de nos besoins, merci.
Vivement 2025 !

**Comment by cpages175** [comment]
Ce serait un plus indéniable ! 
Si l'outil envisagé pouvait être développé autour de la notation BPMn cela amènerait une plus-value encore plus importante à l'usage par les correspondants métier.

**Comment by dbarzin** [comment]
C'est en cours, cela viendra dans les prochains mois.
